tdrs-backend/tdpservice/data_files/admin.py

```diff
@@ -2,13 +2,47 @@
 from django.contrib import admin
 
 from ..core.utils import ReadOnlyAdminMixin
+from django.utils.translation import ugettext_lazy as _
 from .models import DataFile, LegacyFileTransfer
 from tdpservice.parsers.models import DataFileSummary, ParserError
 from django.conf import settings
 from django.utils.html import format_html
 
 DOMAIN = settings.FRONTEND_BASE_URL
 
+
+class NewestVersionFilter(admin.SimpleListFilter):
+    """Simple filter class to show newest created datafile records."""
+
+    title = _('Newest')
```

Comment:
Aren't we trying to change to "Most recent version" ?

Reply:
This changes the "title" of the widget. the value is changed elsewhere. I'd be open to changing the title too, but "Newest" made sense to me here. 

@reitermb i would like your opinion as well

Reply:
@ADPennington @reitermb 
Just hopped off a call with @jtimpe. 

Learning that the way files are actually filtered/sorted on the Search Index versus Data Files using the "Newest" are different, I suggested the following:

On the Search Index Page: Retitle "Newest" To "By Submission" with the dropdown being "Most Recent"
On the Data Files Page: Repurposing the current "By Version" drop down and having the options be "Most Recent" and "All" I noticed that there is a numerical drop down, would we still like to keep that?

<img width="268" alt="version_datafiles" src="https://github.com/user-attachments/assets/f5605426-7c6c-4cda-8ab1-fe5adfa2972d"> 

comment to address the question you had for @reitermb  about the filtering, perhaps the title of the widget is 'File Version' and the default setting from the drop down is "Most Recent"

Reply:
@victoriaatraft @ADPennington i left both in for the time being, but it is a little weird to have two "by version"s. removing the numerical version filter or changing the title of the most-recent widget would be a light lift.

Reply:
@jtimpe also to note, I just realized @raftmsohani is working on #3076  where the "by submission date" filter options on the data files page are: All, Yesterday, Today, Past 7 days, This Month, This Year. 
We may need to revisit the drop down items for By Submission on the search index page to be consistent in items if you think this is valuable in filtering @ADPennington .

<img width="267" alt="Screenshot 2024-08-29 at 10 06 43 PM" src="https://github.com/user-attachments/assets/c0b7e2fe-142c-45ea-b167-649b30cd8ea2">